cpp/daal/src/algorithms/service_kernel_math.h

```diff
@@ -171,6 +173,14 @@ class EuclideanDistances : public PairwiseDistances<FPType, cpu>
         return services::Status();
     }
 
+    virtual services::Status getRootBatch(const size_t n, FPType * in, FPType * out)
```

Comment:
better to align naming across distances
we have init(), we can call this as finalize() or so on

Reply:
applied, thanks for advise